Makefile

```diff
@@ -3,15 +3,15 @@ ci: test
 	npm run docs
 
 test: lint
-	npm run test
+	npm run $@
 
 lint: node_modules
-	npm run lint
+	npm run $@
 
 clean:
 	rm -rf node_modules
 
 node_modules: package.json
 	npm install
 
-.PHONY: node_modules
+.PHONY: clean
```

Comment:
Can you explain why 'clean' is the preferable `.PHONY` target here? The `.PHONY` target forces `make` to execute the target, even if there is an already existing file by that name. Using `node_modules` here means that `npm install` will run even if there is already a `node_modules` directory. Which is what you want in the event that `package.json` has been changed.